src/builders/util.js

```diff
@@ -59,6 +59,12 @@ export function renderSandboxSbConfig(
          (regex "^${config.getRootPath(spec, '.*', '[^/]*\\.install')}$")
         ; $cur__root/NAME.install
          (regex "^${config.getRootPath(spec, '[^/]*\\.install')}$")
+
+        ; $cur__root/NAME.opam
+         (regex "^${config.getRootPath(spec, '[^/]*\\.opam')}$")
+
+        ; $cur__root/jbuild-ignore
+         (regex "^${config.getRootPath(spec, 'jbuild-ignore')}$")
         `
```

Comment:
The other one that I really wanted to add here was `*/jbuild` so that build scripts could auto-generate `jbuild` files, yet still benefit from `buildsInSource: _build`. However, the current whitelist are files that are unlikely to heavily influence the build, whereas `jbuild` files do heavily influence the build. Do you have any suggestions? Perhaps the PR as-is, is as relaxed as we can safely be.